web/server/codechecker_server/api/product_server.py

```diff
@@ -129,6 +129,11 @@ def __get_product(self, session, product):
             confidentiality = \
                     confidentiality_enum(product.confidentiality)
 
+        if product.report_limit is not None:
+            report_limit = product.report_limit
+        else:
+            report_limit = None
```

Comment:
Educate me, why can't we just do this?
```suggestion
        report_limit = product.report_limit
```

Reply:
we can, and we will!